src/libraries/Common/tests/System/Net/Configuration.Http.cs

```diff
@@ -21,7 +21,7 @@ public static partial class Http
 
             // This server doesn't use HTTP/2 server push (push promise) feature. Some HttpClient implementations
             // don't support servers that use push right now.
-            public static string Http2NoPushHost => GetValue("DOTNET_TEST_HTTP2NOPUSHHOST", "www.microsoft.com");
+            public static string Http2NoPushHost => GetValue("DOTNET_TEST_HTTP2NOPUSHHOST", "httpbin.org/get");
```

Comment:
This should be only a hostname, not actual url (i.e. without the /get portion).

the /get portion should be added where the actual uri are constructed although /get path does not make sense for *all* hosts...

Reply:
Done